httpd.go

```diff
@@ -50,7 +53,21 @@ func (h *Httpd) Run() {
 
 func messageHandler(p Param) (int, string) {
 	ch := make(chan error, 1)
-	go MessageBus.Publish(NewMessage(p, ch))
+
+	// The format is compat with JavaScript's date.toISOString().
```

Comment:
I'd been concerned about this point. I will go with unix timestamp because it is one of most popular representation and POST parameter doesn't have to be human readable.